app/views/categories/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<nav class="mycategs">
+  <ul class="categs">
+    <li>
+      <%= link_to '', root_path, class: "arrow-left" %>
+    </li>
+    <li>
+      <h1 class="txt">Categories</h1>
+    </li>
+    <li>
+      <%= button_to 'LOG OUT', destroy_user_session_path, method: :delete, class: "destry" %>
+    </li>
+  </ul>
+</nav>
+<% @categories.each do |category| %>
```

Comment:
- Kindly follow the design guidelines for each category item:
   - [ ] Please change the font color and weight to match the design.
   - [ ] Kindly adjust the date format to match the design.
   - [ ] Kindly increase the spacing between the category title and the date.

| Your design | Guidelines |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/230428000-b5118ccd-1afc-4d11-9f0f-6d34651a3da9.png) | ![image](https://user-images.githubusercontent.com/24703171/220965354-cda3912f-f50e-4f85-ad3e-13358980081c.png) |